movie_api/urls.py

```diff
@@ -0,0 +1,23 @@
+"""move_api URL Configuration
```

Comment:
There is a typo in the project name in the comment. It should be 'movie_api' instead of 'move_api'.